README.md

```diff
@@ -307,7 +307,7 @@ plugins:
 It becomes harder and harder to keep up with YouTube's cipher changes, as they become more frequent and complex.
 To help with this, you can use a remote cipher server to handle signature deciphering for you.
 You can use [yt-cipher](https://github.com/kikkia/yt-cipher), which is a simple Deno server that exposes a REST API for deciphering signatures.
-Check out the repository for more information on how to set it up.
+Check out the repository for more information on how to host it yourself or use their public instance at `https://cipher.kikkia.dev/api/`.
```

Comment:
Not entirely convinced on having third-party services advertised in some form of official capacity, so I'll ask @topi314 for thoughts here as well.